lib/models/project.js

```diff
@@ -569,20 +569,32 @@ Project.closest = function(pathName, _ui, _cli) {
  */
 Project.closestSync = function(pathName, _ui, _cli) {
   var ui = ensureUI(_ui);
-
-  try {
-    var directory = findup.sync(pathName, 'package.json');
-    var pkg = require(path.join(directory, 'package.json'));
-
-    if (pkg && pkg.name === 'ember-cli') {
-      return Project.nullProject(_ui, _cli);
+  var directory, pkg;
+  
+  if (_cli && _cli.testing) {
+    directory = existsSync(path.join(pathName, 'package.json')) && process.cwd();
+    if (!directory) {
+      if (pathName.indexOf('/app') > -1){
```

Comment:
`/app` doesn't work on Windows and seems to cause two failing tests. I've fixed it by using `path.sep + 'app'` instead and will push that solution in a moment.